src/components/ReviewSummary.jsx

```diff
@@ -49,88 +49,122 @@ function ReviewSummary({
-          <div className="pb-6 flex items-center w-full justify-between gap-2 text-xs font-light">
-            <p className="">Too Narrow</p>
+
+          <div className="pb-6 flex items-center w-full justify-between gap-2 text-xs text-neutral-600 font-light">
+            <p>Too Narrow</p>
             <p>Perfect</p>
-            <p>Too Large</p>
+            <p>Too Wide</p>
           </div>
         </div>
-        <div>
-          <h4 className="text-sm">Comfort</h4>
+        <div className="pt-10">
+          <h4 className="text-base pb-2">Comfort</h4>
           <div style={{ position: 'relative' }} className="flex flex-row w-full">
             <span className="pr-2" style={getFeatureData('Comfort')}><TbTriangleInvertedFilled className="text-sm" /></span>
+            <div className="grid grid-cols-5 w-full gap-1 h-2">
+              {Array(5).fill(0).map((item) => <div className="w-full h-full bg-zinc-400" />)}
+            </div>
+
           </div>
-          <progress className="w-full h-2" value={0} />
-          <div className="pb-6 flex items-center w-full justify-between gap-2 text-xs font-light">
-            <p className="">Uncomfortable</p>
-            <p>OK</p>
+
+          <div className="pb-6 flex items-center w-full justify-between gap-2 text-xs text-neutral-600 font-light">
+            <p>Comfort</p>
+            <p>Ok</p>
             <p>Perfect</p>
           </div>
         </div>
-        <div>
-          <h4 className="text-sm">Quality</h4>
+        <div className="pt-10">
+          <h4 className="text-base pb-2">Quality</h4>
           <div style={{ position: 'relative' }} className="flex flex-row w-full">
             <span className="pr-2" style={getFeatureData('Quality')}><TbTriangleInvertedFilled className="text-sm" /></span>
+            <div className="grid grid-cols-5 w-full gap-1 h-2">
+              {Array(5).fill(0).map((item) => <div className="w-full h-full bg-zinc-400" />)}
+            </div>
+
           </div>
-          <progress className="w-full h-2" value={0} />
-          <div className="pb-6 flex items-center w-full justify-between gap-2 text-xs font-light">
-            <p className="">Poor</p>
+
+          <div className="pb-6 flex items-center w-full justify-between gap-2 text-xs text-neutral-600 font-light">
+            <p>Poor</p>
             <p>What I expected</p>
             <p>Perfect</p>
           </div>
         </div>
-        <div>
-          <h4 className="text-sm">Length</h4>
+        <div className="pt-10">
+          <h4 className="text-base pb-2">Length</h4>
           <div style={{ position: 'relative' }} className="flex flex-row w-full">
             <span className="pr-2" style={getFeatureData('Length')}><TbTriangleInvertedFilled className="text-sm" /></span>
+            <div className="grid grid-cols-5 w-full gap-1 h-2">
+              {Array(5).fill(0).map((item) => <div className="w-full h-full bg-zinc-400" />)}
```

Comment:
You've got some mapped elements that don't have keys. Make sure you add those to keep the logs clear.